app/views/docs/webhooks.phtml

```diff
@@ -54,13 +54,101 @@ $events = $this->getParam('events', []);
         </td>
         <td>The ID of the project who owns the Webhook and API call.</td>
     </tr>
+    <tr>
+        <td>
+            X-Appwrite-Webhook-Signature
+            <p class="margin-top-small text-fade">version >= 0.15.0</p>
+        </td>
+        <td>The HMAC-SHA1 signature of the payload. This is used to verify the authenticity of the payload.</td>
+    </tr>
     <tr>
         <td>User-Agent</td>
         <td>Each request made by Appwrite will be 'Appwrite-Server'.</td>
     </tr>
     </tbody>
 </table>
 
+<h2 id="verification"><a href="/docs/webhooks#verification">Verification</a></h2>
+
+<p>Webhooks can be verfied by using the <a href="/docs/webhooks#headers">X-Appwrite-Webhook-Signature</a> header. This is a HMAC-SHA1 
+signature of the payload. You can find the signature key in your webhooks properties in the dashboard. To generate this hash you append
+the payload to the webhook URL and then use the HMAC-SHA1 algorithm to generate the signature.
+
+After you've generated the signature, compare it to the one in the header. If they match, the payload is valid and you can trust it came from
```

Comment:
Replace one with a more specific term
Would it be the hash in the header or the signature in the header or the signed hash in the header?